omod/src/main/java/org/openmrs/web/controller/patient/ShortPatientFormValidator.java

```diff
@@ -89,12 +89,18 @@ public void validate(Object obj, Errors errors) {
 			if (OpenmrsUtil.nullSafeEquals(possibleDuplicate.getId(), personName.getId())) {
 				continue;
 			}
-			
+
 			if (OpenmrsUtil.nullSafeEqualsIgnoreCase(possibleDuplicate.getGivenName(), personName.getGivenName())
-			        && OpenmrsUtil.nullSafeEqualsIgnoreCase(possibleDuplicate.getMiddleName(), personName.getMiddleName())
-			        && OpenmrsUtil.nullSafeEqualsIgnoreCase(possibleDuplicate.getFamilyName(), personName.getFamilyName())) {
-				errors.reject("Patient.duplicateName", new Object[] { personName.getFullName() }, personName.getFullName()
-				        + " is a duplicate name for the same patient");
+					&& OpenmrsUtil.nullSafeEqualsIgnoreCase(possibleDuplicate.getMiddleName(), personName.getMiddleName())
+					&& OpenmrsUtil.nullSafeEqualsIgnoreCase(possibleDuplicate.getFamilyName(), personName.getFamilyName())) {
+					if (possibleDuplicate.isVoided()) {
+						errors.reject("Patient.duplicateName", new Object[]{personName.getFullName()}, personName.getFullName()
```

Comment:
Can you actually use a message code and have this translated in the message.properties file, can you also do the same for the other error message too? Thanks!